### ISSUE TITLE
Let R shiny have chat ui components

### ISSUE THREAD

**Issue by kaipingyang** [jcheng5/r-sidebot#1]
Hi @jcheng5, Thank you for your great work on developing the chat_ui component in r-sidebot, I was wondering if there are any plans for R shiny to have a chat_ui component?

**Comment by jcheng5** [comment]
Yes, we do want to bring it to R officially, although most likely in a separate package from shiny or bslib.

**Comment by jcheng5** [comment]
Now available https://github.com/posit-dev/shinychat and on CRAN
